src/test-utils/index.tsx

```diff
@@ -57,6 +60,22 @@ const DEVICES: DeviceCollection = {
             isLandscape: false,
         },
     },
+    [MOBILE_DEVICE_IOS_INSET]: {
+        userAgent:
+            'Mozilla/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Mobile/15A372 Safari/604.1',
+        platform: 'ios',
+        viewport: {
+            width: 375,
+            height: 667,
+            deviceScaleFactor: 2,
+            isMobile: true,
+            hasTouch: true,
+            isLandscape: false,
+            safeAreaInset: {
+                bottom: '34px',
```

Comment:
this is the inset I got on an iphone 13 mini, I think on a max could be more higher?